packages/sui-pde/src/hooks/useExperiment.js

```diff
@@ -17,6 +17,17 @@ export default function useExperiment(experimentName, attributes) {
 
     try {
       variationName = pde.activateExperiment({name: experimentName, attributes})
+
+      if (variationName && window.analytics?.track) {
+        window.analytics.track('Experiment Viewed', {
```

Comment:
Does it have the sense to use `window.sui.analytics` in order to get the segment wrapper benefits?
May could be a way to pass a custom callback or sth else to make this track? I mean, doing `window.analytics.track` I think that we are making this package less "open source"

Reply:
sui.analytics shouldn't be an issue as the segment wrapper does patch window.analytics https://github.mpi-internal.com/scmspain/frontend-all--segment-wrapper/blob/master/src/patchAnalytics.js

regarding the callback question, yep you are right although I personally prefer adding it once someone requests it instead of having code that actually no one uses. Maybe it's something to discuss within the team, what do we wan't to do in these cases? What should be our approach from now on?